### PR TITLE
chrome: Really fix chrome.webNavigation event filtering

### LINKED ISSUE #14763: Incorrect Chrome definitions
[This](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/df23346707493018518e7c8cd04d7e6e7dce1f05) commit removed 'url' from chrome.webNavigation filters which is valid according to the spec:

https://developer.chrome.com/extensions/webNavigation
https://developer.chrome.com/extensions/events#filtered
```js
      chrome.webNavigation.onCommitted.addListener(function(e) {
        // ...
      }, {url: [{hostSuffix: 'google.com'},
                {hostSuffix: 'google.com.au'}]});
      
```

https://chromium.googlesource.com/chromium/chromium/+/master/chrome/common/extensions/api/web_navigation.json

  - Authors:  @sreimer15

### PR DESCRIPTION
**Pull request by ghost**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [event filtering docs](https://developer.chrome.com/extensions/events#filtered)
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

Closes https://github.com/DefinitelyTyped/DefinitelyTyped/issues/14763 and https://github.com/DefinitelyTyped/DefinitelyTyped/issues/14330

I wrote a boring wall of text here but realized no one was going to read it. Maybe see my other comment [here](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/14330#issuecomment-289187509) if you really want to read something about this PR. It's confusing because I was confused too.

TL;DR: A previous commit https://github.com/DefinitelyTyped/DefinitelyTyped/commit/df23346707493018518e7c8cd04d7e6e7dce1f05 mistakenly broke the types for filtering `chrome.webNavigation` events, and this one fixes them.

### CODE REVIEW

**Comment by epicgirl1998** [comment]
i will now proceed to beat myself up for not better at explaining things without being confusing

this pull was meant to be a no-brainer merge like the previous one i made that was accepted within the same day

im sorry

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -7189,7 +7190,7 @@ declare namespace chrome.webRequest {
          */
         types?: string[];
         /** A list of URLs or URL patterns. Requests that cannot match any of the URLs will be filtered out. */
-        urls: string[] | RegExp[] | "<all_urls>";
+        urls: string[];
```
Comment (NN---) [inline types/chrome/index.d.ts]: It should be 'url' not 'urls'.
See: https://developer.chrome.com/extensions/events

**Review by NN--- — COMMENTED** [review]
Incorrect definition

**Comment by epicgirl1998** [comment]
that's the filter type for the `chrome.webRequest` api which does use `urls` (array of strings)

https://developer.chrome.com/extensions/webRequest#examples

it was changed to `string[] | RegExp[] | "<all_urls>"` in https://github.com/DefinitelyTyped/DefinitelyTyped/commit/df23346707493018518e7c8cd04d7e6e7dce1f05 even though it should be just `string[]` so i changed it back

**Comment by NN---** [comment]
Oh sorry, I missed WebRequest part
